src/components/Button.js

```diff
@@ -1,8 +1,13 @@
 import './../styles/Button.css';
+import Button from '@mui/material/Button';
 
-function Button(props) {
+export default function BasicButtons(props) {
     let defaultOnClick = () => alert("Not yet implemented")
-    return <button className='button' onClick={props.onClick ?? defaultOnClick} disabled={props.disabled}>{props.text}</button>;
-}
-
-export default Button;
\ No newline at end of file
+    return (
+    <Button 
+    style={{
+        backgroundColor: "#2277d9",
```

Comment:
Remove this and use proper styling with classes/CSS files so that all our styling is in one place, and there's no "Why are these buttons still blue???" moments.